packages/opentelemetry-core/src/common/time.ts

```diff
@@ -175,3 +166,14 @@ export function isTimeInput(
     value instanceof Date
   );
 }
+
+export function addHrTimes(time1: api.HrTime, time2: api.HrTime): api.HrTime {
+  const out: api.HrTime = [time1[0] + time2[0], time1[1] + time2[1]];
+
+  if (out[1] > SECOND_TO_NANOSECONDS) {
+    out[0] = out[0] + Math.floor(out[1] / SECOND_TO_NANOSECONDS);
+    out[1] = out[1] % SECOND_TO_NANOSECONDS;
+  }
```

Comment:
Dirft can easily be hours to days, one second isn't enough

Reply:
Oh this might just be leftover from [previous PR where offset was only put into 2nd element](https://github.com/open-telemetry/opentelemetry-js/pull/3384/files#diff-302359e2304631b0bcd722da02100bfcbd6847ef288ccf5344b885f83c6821b4R247), seems like yeah in this PR it  doesn't have such use case

Reply:
@t2t2 is exactly right it was about defending against the second element summing to more than 2 seconds. I can remove it.